src/wasm2js.h

```diff
@@ -203,27 +221,18 @@ class Wasm2JSBuilder {
       }
       auto mangled = asmangle(out.str());
       ret = stringToIString(mangled);
-      if (!allMangledNames.count(ret)) {
+      if (!set.count(ret) && !(additionalSet && additionalSet->count(ret))) {
```

Comment:
I think this would be easier to read if the `!` were distributed inside the parens.